python/interpret_text/common/preprocessor/glove_preprocessor.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import pandas as pd
+from typing import Any, Iterable, Tuple, List
+from interpret_text.common.base_explainer import BaseTextPreprocessor
+
+
+class GlovePreprocessor(BaseTextPreprocessor):
+
+    def __init__(self, count_threshold: int, token_cutoff: int):
+        self.count_threshold = count_threshold
+        self.token_cutoff = token_cutoff
+
+    def build_vocab(self, text) -> Any:
+        words_to_indexes = {"<PAD>": 0, "<UNK>": 1}
+        counts = {}
+        for sentence in text:
+            for word in sentence.split():
+                if word not in words_to_indexes:
+                    words_to_indexes[word] = len(words_to_indexes)
+                    counts[word] = 1
+                else:
+                    counts[word] += 1
+        indexes_to_words = {v: k for k, v in words_to_indexes.items()}
+
+        self.word_vocab = words_to_indexes
+        self.reverse_word_vocab = indexes_to_words
+        self.counts = counts
+
+    def preprocess(self, data) -> Iterable[Any]:
+        token_lists = []
+        masks = []
+        counts = []
+        for sentence in data:
+            token_list, mask = self.generate_tokens(sentence)
+            token_lists.append(token_list)
+            masks.append(mask)
+            counts.append(np.sum(mask))
+        tokens = pd.DataFrame(
+            {"tokens": token_lists, "mask": masks, "counts": counts}
```

Comment:
this seems to have a common structure with other class.  I wonder if we should have a more explicit representation for this.